hs_labels/views/__init__.py

```diff
@@ -6,9 +6,10 @@
 from hs_core.views import authorize
 
 
-def resource_labeling_action(request, shortkey, *args, **kwargs):
+def resource_labeling_action(request, shortkey=None, *args, **kwargs):
     """
     This must be a POST request. The following data needs to be passed as part of the POST request:
+	shortkey: Id of the resource (Must be None when the label_type is 'SAVEDLABEL')
     action: resource labeling action - has to be either 'CREATE' or 'DELETE'
     label_type: type of label - one of these: 'LABEL', 'FAVORITE', 'MINE', 'FOLDER', 'SAVEDLABEL'
```

Comment:
Actually, it is not being used. But the API is being called to store the information. The folder API is subject to (temporary) deprecation. I am going to approve this anyway on the grounds that there is "no harm done" but in the future, we should probably remove references to the folder API.